libs/core/src/components/sage-tabs/sage-tabs.tsx

```diff
@@ -0,0 +1,161 @@
+import { Component, Element, Host, h, Prop, Listen } from '@stencil/core';
+
+@Component({
+  tag: 'sage-tabs',
+  styleUrl: 'sage-tabs.scss',
+  shadow: true,
+})
+export class SageTabs {
+  private tabs;
+  private tabPanels;
+  @Element() el: HTMLDivElement;
+
+  /**
+   * Sets the aria-label attached to the tablist element, required
+  */
+  @Prop() tablistLabel: string;
+ 
+  /**
+    * Sets unique id on tabs component, required
+  */
+  @Prop() componentId: string;
+  
+  /**
+   * Sets tabs variant styles as outlined in Figma documentation, optional
+   * @defaultValue primary
+    */
+  @Prop() variant?: 'primary' | 'availability' | 'filter' = 'primary';
+
+  /**
+   * Sets default active tab, required
+   */
```

Comment:
Default and active together are feel a little weird since it changes based on user feedback